attributes/storm.rb

```diff
@@ -0,0 +1,49 @@
+# storm release
+default['storm']['release']['install']       = false
+default['storm']['release']['install_path']  = '/opt'
+default['storm']['release']['version']       = '0.9.5'
+default['storm']['release']['checksum']      = '2e8337126de8d1e180abe77fb81af7c971f8c4b2dad94e446ac86c0f02ba3fb2'
+# default['storm']['release']['version']       = '0.10.0-beta1'
+# default['storm']['release']['checksum']      = 'e54b400b1e1a012149e74602e2441b355c433f27773e7191172342c8b595467d'
```

Comment:
Turn checksum into a case, so you can have both versions without requiring overriding everything, such as in the [CDAP cookbook](https://github.com/caskdata/cdap_cookbook/blob/master/attributes/sdk.rb#L24-L48)...